packages/slate/src/changes/at-range.js

```diff
@@ -863,7 +863,7 @@ Changes.insertTextAtRange = (change, range, text, marks, options = {}) => {
   }
 
   // PERF: Unless specified, don't normalize if only inserting text.
```

Comment:
Got it, I added the getFlag part.
Do we really want to ignore an explicitly set `{ normalize: true }` in the function call though?
It sounds counter-intuitive to me.
An additional call to `change.normalizeNodeByKey(change.value.startBlock.key)` would be required to do the trick, but would users know?

I'll change the condition if you confirm.

Reply:
Indeed, we are trying to normalize the `change.deleteAtRange` in the insertTextAtRange function... That is why we need such a complex normalization.

Reply:
@zhujinxuan what's the reason for doing so? 
Prevent calling normalization twice for performance reasons?

Reply:
Because both `insertTextAtRange` (though bugy right now) and `insertTextByKey` are public interface, we want the same performance optimization exist in both functions for someone calling it from outside.

Reply:
But why couldn't we move the optimisation code to `deleteAtRange` then?
Plus I don't see a lot of common normalization code between `insertTextAtRange` and `insertTextByKey`